src/nlp_sdg/pipelines/data_engineering/pipeline.py

```diff
@@ -14,10 +14,10 @@ def create_pipeline(**kwargs) -> Pipeline:
         
         [
             node(
-                func=dummy_node,
+                func=preprocess_data,
                 inputs="sdg_data",
-                outputs="model_input_data",
-                name="dummy_node",
+                outputs="preprocess_data",
+                name="preprocess_data",
```

Comment:
@JayHansea I don't see the changes made to the DataCatalog for the output on this node, Can you please add this to the DataCatalog. Can we please call the output preprocess**ed**_data (past-tense)